pyuvdata/uvbeam/tests/test_cst_beam.py

```diff
@@ -49,6 +49,13 @@ def cst_efield_2freq_mod(cst_efield_2freq_main):
     del beam
 
 
+def test_swapped_cst_beam():
+    beam = UVBeam()
+    beam.read_cst_beam(
+        os.path.join(DATA_PATH, cst_folder, "CST_f50_Eloy.txt"),
+    )
```

Comment:
Should we check anything about this beam object? Maybe that the value in a particular spot is what we expect it to be or that the peak is at zenith? So we're sure it was actually read in properly.

Reply:
I updated it to check whether the amplitudes at zenith/anti-zenith are all the same.